Modules/CodeTrek/Emails/CodetrekMailApplicant.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Modules\CodeTrek\Emails;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+
+class CodetrekMailApplicant extends Mailable
+{
+    use Queueable, SerializesModels;
+    public $data;
+
+    /**
+     * Create a new message instance.
+     *
+     * @return void
+     */
+    public function __construct($data)
```

Comment:
Please update all the places accordingly.
```suggestion
    public function __construct($Applicant)
```